### PR TITLE
Update for API 9 plus other features and fixes (all changes combined)

### PR DESCRIPTION
**Pull request by nebel**
As discussed, this PR includes all changes from:

- #60 
- #56 
- #57 
- #59 

If this is merged, all the above PRs can be closed.

If any further changes are required to the above PRs, I can just put them here for simplicity's sake (if that is preferred).

### CODE REVIEW

File: Dalamud.FindAnything/Input.cs [hunk 1]
```diff
@@ -40,7 +39,7 @@ public static bool IsGameFocused
     public Input()
     {
         try { isTextInputActivePtr = *(IntPtr*)((IntPtr)AtkStage.GetSingleton() + 0x28) + 0x188E; } // Located in AtkInputManager
```
Comment (goaaats) [inline Dalamud.FindAnything/Input.cs:41]: I bet this is in CS now, if it isn't we should totally go and add it

**Review by goaaats — APPROVED** [review]

**Comment by goaaats** [comment]
Looks great! Thanks so much, this is amazing.

File: Dalamud.FindAnything/Input.cs (hunk 1 above)
Comment (nebel) [reply Dalamud.FindAnything/Input.cs:41]: Looks like this exact code is also used in InventorySearchBar and TPie, so might be useful to have in CS...

I did notice that QoLBar now uses different logic:
```public static bool IsGameTextInputActive => uiModule->GetRaptureAtkModule()->AtkModule.IsTextInputActive();```

Not sure if what differences there might be between these two methods, though.

**Comment by nebel** [comment]
With all the work on open with double tap seemed like I accidentally broke open with combo. The commit I pushed just now should fix that.

File: Dalamud.FindAnything/Input.cs (hunk 1 above)
Comment (nebel) [reply Dalamud.FindAnything/Input.cs]: Switched to using QoL Bar's method in the latest commit.

**Review by goaaats — APPROVED** [review]

**Pull request merged**